tests/unit/models/item-test.js

```diff
@@ -59,3 +59,102 @@ test('allPackageCategories', function(assert) {
 
   assert.equal(record.get('allPackageCategories.firstObject.id'), [pkgCategory.id]);
 });
+
+
+test('isItem', function(assert){
```

Comment:
Can we give some proper description for the test?

Reply:
Sure 👍